easybuild/easyconfigs/l/LMfit/LMfit-1.0.0-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'PythonBundle'
+
+name = 'LMfit'
+version = '1.0.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://lmfit.github.io/lmfit-py'
+description = """Lmfit provides a high-level interface to non-linear optimization
+and curve fitting problems for Python"""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('asteval', '0.9.17', {
```

Comment:
@ekaterinaasf The existing `LMfit` easyconfig using `intel/2019b` as toolchain uses `asteval` 0.9.18, so I see no reason not to do that here too?